packages/allure-js-commons/src/sdk/reporter/types.ts

```diff
@@ -86,9 +86,9 @@ export interface Writer {
 
   writeGroup(result: TestResultContainer): void;
 
-  writeAttachment(distFileName: string, content: Buffer | string, encoding?: BufferEncoding): void;
+  writeAttachment(distFileName: string, content: Buffer): void;
 
-  writeAttachmentFromPath(from: string, distFileName: string): void;
+  writeAttachmentFromPath(distFileName: string, from: string): void;
```

Comment:
`to` will be much easier to understand what the argument does

Reply:
It's just aligned with the `writeAttachment` method. We can revisit the naming in separate PR